src/providers/rpc/index.ts

```diff
@@ -129,4 +131,23 @@ export class RPCProvider implements IRPCProvider {
         root: result.root || null,
       }));
   }
+
+  public getBlockByHash(txhash: string): Promise<IBlock> {
+    return this.client
+      .call(this.requests.getBlockByHash(txhash))
+      .then(isValidBlockByHash)
+      .then(({ result }) => ({
+        ...result,
+        number: result.number ? hexToNumber(result.number) : null,
+        difficulty: hexToNumber(result.difficulty),
+        totalDifficulty: hexToNumber(result.totalDifficulty),
+        size: hexToNumber(result.size),
+        gasLimit: hexToNumber(result.gasLimit),
+        gasUsed: hexToNumber(result.gasUsed),
+        timestamp: hexToNumber(result.timestamp),
+        minimumGasPrice: result.minimumGasPrice
```

Comment:
Can just leave this as 
```ts
minimumGasPrice: result.minimumGasPrice
```
since if it's undefined then we just leave it as so

Reply:
Explicit property assignment can be omitted, right?